src/sentry/sentry_apps/api/serializers/servicehookproject.py

```diff
@@ -0,0 +1,11 @@
+from sentry.api.serializers import Serializer, register
+from sentry.sentry_apps.models.servicehook import ServiceHookProject
+
+
+@register(ServiceHookProject)
+class ServiceHookProjectSerializer(Serializer):
+    def serialize(self, obj, attrs, user, **kwargs):
+        return {
+            "id": obj.id,
+            "project_id": obj.project_id,
+        }
```

Comment:
I'm not sure what the frontend will need here, but I'm not totally sure if this is enough for a ui

Reply:
is it okay if we leave this as is for now?